### PR TITLE
Removed icons in confirmation buttons to match the other elements in UI

### LINKED ISSUE #2699: Floppy Disk Symbol in "OK"/"Add" Button When Adding New Wire Graph Component or New Channel
When adding new wire graph component, there is a floppy disk symbol next to it (see screenshot bellow), but this is not unified with the rest of the UI - only when adding new channel in "Drivers and Assets" there is a floppy disk symbol next to "Add" button, all other buttons do not have any symbol. 
I suggest this gets changed so the whole UI will get a unifying look. 

**Testflow:**
1. Login to Kura as admin/admin
2. Go to Wires, click on any of the wire components and observe the "OK" button - it has a floppy disk symbol next to it
3. Go to Drivers and Assets, create new drier, then a new asset and then add new channel to the asset - observe the floppy disk symbol in the "Add" button

**Expected behavior**
All of the buttons should have some symbols on them or none of them. 

**Screenshot No. 1 (Floppy disk symbol in Wire graph):**
Screenshot 2020-03-10 at 08 29 31

**Screenshot No. 2 (Floppy disk symbol when adding new channel in "Drivers and Assets"):**
Screenshot 2020-03-10 at 08 23 35

Target Environment (please complete the following information):
Board: Raspberry Pi 3
OS version: Linux raspberrypi 4.19.75-v7+ # 1270 SMP Tue Sep 24 18:45:11 BST 2019 armv7l GNU/Linux
Additional info: /

### PR DESCRIPTION
**Pull request by MMaiero**
Signed-off-by: Maiero <matteo.maiero@eurotech.com>

Closes #2699

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/drivers/assets/AssetConfigurationUi.ui.xml [hunk 1]
```diff
@@ -86,9 +86,9 @@
                     </b:Form>
                 </b:ModalBody>
                 <b:ModalFooter>
-                    <b:Button b:id="btn-create-channel-cancel" icon="REMOVE" type="PRIMARY" dataDismiss="MODAL"
+                    <b:Button b:id="btn-create-channel-cancel" type="PRIMARY" dataDismiss="MODAL"
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/drivers/assets/AssetConfigurationUi.ui.xml:89]: Please update the year in th header to 2020.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/WiresDialogs.ui.xml [hunk 2]
```diff
@@ -144,9 +144,9 @@
                 </b:Form>
             </b:ModalBody>
             <b:ModalFooter>
-                <b:Button b:id="btn-create-comp-cancel" icon="REMOVE" type="PRIMARY" dataDismiss="MODAL"
+                <b:Button b:id="btn-create-comp-cancel" type="PRIMARY" dataDismiss="MODAL"
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/WiresDialogs.ui.xml:147]: Please update the year in th header to 2020.

**Review by LeoNerdoG — CHANGES_REQUESTED** [review]
Hey @MMaiero,
I have checked this PR and it looks OK - after updating the headers this can be merged into develop.

**Comment by MMaiero** [comment]
Done

**Review by LeoNerdoG — APPROVED** [review]

**Pull request merged**